src/stop_motion_obj.py

```diff
@@ -430,13 +447,13 @@ def deleteLinkedMeshMaterials(mesh, maxMaterialUsers=1, maxImageUsers=0):
     mesh.materials.clear()
 
 
-def newMeshSequence():
+def newMeshSequence(meshName):
```

Comment:
What is gained by being able to specify the name of this empty mesh?

Reply:
It's just a minor UI tweak, as otherwise, when imported in Single Mesh Mode, the mesh container would always be called "emptyMesh" as the container never gets swapped, and therefore the name stays the same.

I found it a bit irritating, but there is no real purpose besides avoiding confusion for the user. I'm totally fine if you would like to remove this!

Reply:
You can leave it in. That's fine